.github/workflows/publish.yml

```diff
@@ -5,29 +5,66 @@ on:
     - 'v0.5.*'
   workflow_dispatch:
   schedule:
-  # Snapshot release every second work day
-  - cron: '0 0 * * 1,3,5'
+  # Snapshot release every day at midnight
+  - cron: '0 0 * * *'
 
 jobs:
-  check-compiles:
+  check_compiles:
     name: Test compilation of all modules
     runs-on: ubuntu-22.04
     if: github.repository == 'scala-native/scala-native'
     steps:
       - uses: actions/checkout@v5
+
+      - name: Restore previous SHA
```

Comment:
As you suggested elsewhere when I mentioned that git is moving away from
SHA-1, perhaps 'gitref`? 'commitHash' is not a term of art, and so adds obfuscation
but in might be useful.